_posts/2016/2016-05-13-geecon.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "GeeCON Kraków"
+image: blog-header/geecon.png
+date: "2016-05-13 07:16:34"
+categories: events
+authors: ["Mathias"]
+---
+
+Sometimes it is a good idea as a developer to step back and do something different for a few days. So you can broaden your view and reflect on your work. Visiting a conference is a good chance to do so - you can learn about new technology, listen to people who question things that you considered a given and see how other people solve similar problem.
+
+
+It comes in handy that epages offers its developers to go to conferences to take advantage of all these things. I took the chance and decided to visit the [GeeCON 2016 in Kraków](http://2016.geecon.org/).
+
+The GeeCON is a three-day conference that focusses on JVM technology and targets mostly the Polish developer community. It is organised by a team of ten people that do this in their spare time besides their main job.
+
+Similar to the Code Talks in Hamburg the conference took place in a cinema. This venue offered giant screens and cozy seats. Three different tracks were offered and the variety of talks covered anything that the industry currently talks about - microservices, reactive applications, frameworks like Akka, Ratpack, Play and Spring Boot, JVM languages like Scala, Clojure, Kotlin and of course Java.
+
+Here is a list of my personal highlights:
+
+## Josh Long - "Cloud Native Java"
+
+Josh Long ([@starbuxman](https://twitter.com/starbuxman)) is the spring developer advocate that travels the world to spread the word about the greatness of spring boot to implement *Cloud Native Applications*. This is another - maybe better suited term - for microservices. At epages we work a lot with Spring Cloud and Spring Boot so most of the content was not new to me. The interesting part was rather that Josh is the master of live-coding. He does that at an incredible speed and with a great level of entertainment. So that was really a show.
```

Comment:
...is THE Spring developer advocate, and travels the world to spread the ... of Spring Boot ...

> would prefer Spring Boot with upper case letters, see here: http://projects.spring.io/spring-boot/

At ePages, we work a... Spring Boot, so most of (add commas)